.eslintrc.js

```diff
@@ -142,6 +142,7 @@ const baseConfig = {
         // prefer TypeScript exhaustiveness checking
         // https://www.typescriptlang.org/docs/handbook/advanced-types.html#exhaustiveness-checking
         'default-case': OFF,
+        'import/order': [ERROR, { 'newlines-between': 'always' }],
```

Comment:
A bit more testing on the frontend led me to:

```suggestion
        'import/order': [
          ERROR,
          {
            alphabetize: {
              order: 'asc',
            },
            'newlines-between': 'always',
            pathGroups: [
              {
                pattern: 'src/**',
                group: 'external',
                position: 'after',
              },
            ],
            pathGroupsExcludedImportTypes: [
              {
                pattern: 'src/**',
                group: 'external',
                position: 'after',
              },
            ],
          },
        ],
```

Which is scary (the `pathGroups` don't seem to work very intuitively) but nets


```typescript
// doesn't touch this
import 'braid-design-system/reset';

import { BraidLoadableProvider } from 'braid-design-system';

import { Header } from 'src/components/Header';

import * as styleRefs from './App.treat.ts';
```

Reply:
Was looking at this as well. Not sure `pattern: 'src/**'` makes sense though as not all projects use `src` and some have other top-level folders.